lib/prawn/table/cell.rb

```diff
@@ -717,8 +716,13 @@ def draw_borders(pt)
         x, y = pt
 
         @pdf.mask(:line_width, :stroke_color) do
+          pad_top    = @borders.include?(:top)    && @border_widths[0] > 0 && @border_lines[0] == :double && 1 || 0
```

Comment:
If you move the `idx` naming logic from the loop below here or to a constant you'd be able to do these 4 in a loop as well.

I don't care that much but it would be more consistent if we stick with one thing throughout the method. Either always use _magic numbers_ or always use symbols that translate to these numbers.